src/hooks/useAppInsights.js

```diff
@@ -21,6 +21,12 @@ export default function useAppInsights() {
     ai.loadAppInsights();
     ai.addTelemetryInitializer((item) => {
       item.tags['ai.cloud.role'] = 'SSW.Rules-StaticClientPage';
+      if (
+        item.baseData?.target?.includes('analytics.google.com') &&
+        item.baseData?.responseCode == 0
+      ) {
+        return false; // Exclude Google Analytics logs with responseCode=0
```

Comment:
As per our conversation with @Calinator444 

1. Track these as successes not failures - this way they are still in app insights
2. Add a link to your investigation in the code

Add the 
```suggestion
        return false; // Exclude Google Analytics logs with responseCode=0 https://github.com/SSWConsulting/SSW.Rules/issues/1589#issuecomment-2437107468
```

Reply:
✅ Done